### ISSUE TITLE
Search on ollama.com/library is missing lots of models

### ISSUE THREAD

**Issue by maxtheman** [ollama/ollama#3007]
Current behavior:

Using @ehartford as an example since he's a prolific ollama model contributor:

https://ollama.com/search?q=ehartford&p=1

Shows his models:

<img width="1276" alt="Screenshot 2024-03-08 at 9 45 33 AM" src="https://github.com/ollama/ollama/assets/2172753/08b9dc80-5d94-4b86-82dd-37c0dddac326">

https://ollama.com/library?q=ehartford

Does not:

<img width="1257" alt="Screenshot 2024-03-08 at 9 45 52 AM" src="https://github.com/ollama/ollama/assets/2172753/6f1c248b-a93c-468a-a78c-24811a857731">

Expected behavior is that they would be the same.

**Comment by hoyyeva** [comment]
Hi @maxtheman 

Recently, we cleaned up the site and spent time improving the user experience, especially with searching on Ollama. I hope the issue has been resolved. However, if you’re still encountering any confusion or difficulty navigating the site, please feel free to reopen the issue.

**Comment by maxtheman** [comment]
@hoyyeva Yes, the new site looks great and I am quite impressed that you managed to track down and close an issue from almost a year ago, genuinely impressive project management. Keep up the great work!
